src/Stepper/Step.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Step extends Component {
+  constructor() {
+    super();
+    this.getStyles = this.getStyles.bind(this);
+  }
+
+  getStyles() {
+    const {
+      activeColor, completeColor, defaultColor, circleFontColor,
```

Comment:
Instead of making all these specific props why not let them do things like `stepStyle`, `circleStyle`, etc. That way any style prop is override-able.